tests/io_perfomance/image-volume/run-iostat.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+#Running IOSTAT
+for (( i = 1; i <= 36; i++ ))
+do
+iostat -xm 3 20 > ~/"$GIT_REPO"/"$FOLDERNAME"/Configs/Test-results/Iostat/iostat-fio-group-"$i"
```

Comment:
iostat for now is fine @vk-en but in the long run we will probably need to develop side car performance/observability containers for this kind of a task.

Reply:
We have 36 fio tests in total, so we create separate statistics for each Fio test, collected from Iostat. This way we get more accurate performance data for each Fio test, and not for one file, in which it is difficult to understand in which test we got this or that result.
If the number of tests increases, this range will need to be expanded.